app/components/Settings/Settings.react.js

```diff
@@ -146,7 +156,17 @@ export default class Settings extends Component {
                     <h4>
                         Plotly 2.0 Database Connector
                     </h4>
-
+                    <div className={styles.info}>
+                        This application is running on {" "}
+                        <u>http://localhost:{OPTIONS.port}</u> and {" "}
+                        <u>https://connector.plot.ly:{OPTIONS.port + 1}</u>
```

Comment:
nice, I didn't actually know about `<u>`. This went [in and out of deprecation over the years](http://html5doctor.com/u-element/). Totally valid for us to use now but we usually try to reserve underlines for links. Would `<b>` work instead here?